app/views/insured/group_selection/_coverage_household.html.erb

```diff
@@ -9,7 +9,7 @@
     <%# show_residency_alert = false %>
     <% @coverage_household.coverage_household_members.map(&:family_member).each_with_index do |family_member, index| %>
         <% if @benefit_group.sole_source? %>
-        <% is_coverage = offered_relationship_benefits.present? ? composite_relationship_check(offered_relationship_benefits, family_member, @new_effective_on) : true
+        <% is_coverage = offered_relationship_benefits.present? ? composite_relationship_check(offered_relationship_benefits, family_member, @new_effective_on, coverage_kind) : true
```

Comment:
I think if we changed the line https://github.com/health-connector/enroll/blob/55f9c64f0b1731a592aa2181ae6e4202099b6afd/app/views/insured/group_selection/_coverage_household.html.erb#L11
to: 
`        <% if @benefit_group.sole_source? && coverage_kind == 'health' %>
`
we can accomplish the same behavior without needing any changes in composite_relationship_check